### PR TITLE
Initial support for v1.19.30

### PR DESCRIPTION
**Pull request by JustTalDevelops**

### CODE REVIEW

**Comment by JustTalDevelops** [comment]
This is ready for review now.

File: go.mod [hunk 1]
```diff
@@ -24,3 +25,5 @@ require (
 	google.golang.org/appengine v1.6.6 // indirect
 	google.golang.org/protobuf v1.25.0 // indirect
 )
+
+replace github.com/sandertv/go-raknet => github.com/justtaldevelops/go-raknet v0.0.0-20220912200218-80153477b5b0
```
Comment (Sandertv) [inline go.mod:29]: Don't forget to correct this. sandertv/go-raknet now has v1.12.0 which has the right protocol ver.

File: minecraft/protocol/packet/login.go [hunk 2]
```diff
@@ -7,8 +7,9 @@ import (
 // Login is sent when the client initially tries to join the server. It is the first packet sent and contains
 // information specific to the player.
 type Login struct {
-	// ClientProtocol is the protocol version of the player. The player is disconnected if the protocol is
-	// incompatible with the protocol of the server.
+	// ClientProtocol is the protocol version of the player. The player is disconnected if the protocol is incompatible
+	// with the protocol of the server. It is unclear why this is still sent in the login packet, as the ClientProtocol
+	// is already sent in the RequestNetworkSettings packet.
```
Comment (Sandertv) [inline minecraft/protocol/packet/login.go:12]: I wouldn't say it's unclear, I would just say it has been superseded by the protocol version sent in the RequestNetworkSettings packet and that this should no longer be used by the server.

File: minecraft/protocol/packet/pool.go [hunk 3]
```diff
@@ -215,6 +215,10 @@ func init() {
 		IDDeathInfo:                         func() Packet { return &DeathInfo{} },
 		IDEditorNetwork:                     func() Packet { return &EditorNetwork{} },
 		IDFeatureRegistry:                   func() Packet { return &FeatureRegistry{} },
+		IDServerStats:                       func() Packet { return &ServerStats{} },
+		IDRequestNetworkSettings:            func() Packet { return &RequestNetworkSettings{} },
+		IDGameTestRequest:                   func() Packet { return &GameTestRequest{} },
+		// TODO: Game test results
```
Comment (Sandertv) [inline minecraft/protocol/packet/pool.go:221]: Don't forget to resolve this.

**Review by Sandertv — COMMENTED** [review]
Got a couple of minor comments/reminders.

**Pull request merged**